blog-api/src/Queue/TestHandler.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Queue;
+
+use Psr\Log\LoggerInterface;
+use Yiisoft\Json\Json;
+use Yiisoft\Yii\Queue\Message\Message;
+
+class TestHandler
+{
+    public const NAME = 'test-handler';
+    public const CHANNEL = 'channel-1';
+
+    public function __construct(private LoggerInterface $logger)
+    {
+    }
+
+    public function handle(Message $message): void
+    {
+        #StreamTarget doesn't log to stdout with queue/listen ??
```

Comment:
It does, but this case needs some more logger configuration. By default it accumulates a thousand messages before writing them to any target. Here is an example config:
```php
    LoggerInterface::class => [
        'class' => Logger::class,
        '__construct()' => [
            'targets' => ReferencesArray::from([
                FileTarget::class,
                'target-stdout',
            ]),
        ],
        'setFlushInterval()' => [1],
    ],

    'target-stdout' =>                 [
        'class' => StreamTarget::class,
        'setExportInterval()' => [1],
    ],
```

It's very complicated and unintuitive, yes, and that's subject to change in future versions.